arcade/arcade/sdk/eval/eval.py

```diff
@@ -474,6 +480,54 @@ def add_case(
         )
         self.cases.append(case)
 
+    def _add_none_critics(
```

Comment:
Feature `2` from PR description, along with changes to `display.py` and `critic.py`